assets/aircraft/js32.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "British Aerospace BAe-3200 Jetstream Super 31",
+    "icao": "JS32",
+    "engines": {
+        "number": 2,
+        "type": "S"
+    },
+    "weightClass": "M",
+    "category": {
+        "srs": 3,
+        "lahso": 5,
+        "recat": "F"
+    },
+    "ceiling": 25000,
+    "rate": {
+        "climb":      2000,
+        "descent":    2000,
+        "accelerate": 3,
+        "decelerate": 4
+    },
+    "runway": {
+        "takeoff": 1.570,
+        "landing": 1.220
+    },
+    "speed":{
+        "min":     86,
+        "landing": 115,
+        "cruise":  235,
```

Comment:
The cruise speed at [eurocontrol](https://contentzone.eurocontrol.int/aircraftperformance/details.aspx?ICAO=JS32&ICAOFilter=js32) is in True Airspeed while the other are in Indicated Airspeed, is that okay?